me.kozec.syncthingtk.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="windows-1252"?>
+<component type="desktop">
+  <id>syncthing-gtk.desktop</id>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>GPL-2.0</project_license>
+  <name>Syncthing-GTK</name>
+  <summary>GTK3 &amp; Python based GUI and notification area icon for Syncthing</summary>
+  <description>
+    <p>GTK3 &amp; Python based GUI and notification area icon for Syncthing.</p>
+    <p>Supported Syncthing features:</p>
+    <ul>
+      <li>Everything what WebUI can display</li>
+      <li>Adding / editing / deleting nodes</li>
+      <li>Adding / editing / deleting repositories</li>
+      <li>Restart / shutdown server</li>
+      <li>Editing daemon settings</li>
+    </ul>
+    <p>Additional features:</p>
+    <ul>
+      <li>First run wizard for initial configuration</li>
+      <li>Running Syncthing daemon in background</li>
+      <li>Half-automatic setup for new nodes and repositories</li>
+      <li>Filesystem watching and instant synchronization using inotify</li>
+      <li>Nautilus (a.k.a. Files), Nemo and Caja integration</li>
+      <li>Desktop notifications</li>
+    </ul>
+  </description>
+  <launchable type="desktop-id">syncthing-gtk.desktop</launchable>
+  <provides>
+    <binary>syncthing-gtk</binary>
+  </provides>
+  <update_contact>decathorpe_AT_fedoraproject.org</update_contact>
+  <url type="homepage">https://github.com/syncthing/syncthing-gtk</url>
+  <url type="bugtracker">https://github.com/syncthing/syncthing-gtk/issues</url>
+</component>
```

Comment:
The file misses release section, OARS, screenshots and has not been submitted upstream.